libraries/JGUZDV.Blazor.Components.L10n/src/JGUZDV.Blazor.Components.L10n.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+
+      <PackageId>JGUZDV.Blazor.Components.L10n</PackageId>
+      <IsPackable>true</IsPackable>
+      <Version>1.0.0</Version>
+      <Authors>Aylin Siebel</Authors>
+      <Company>Zentrum für Datenverarbeitung - JGU Mainz</Company>
+      <PackageDescription>Blazor component to handle L10nString</PackageDescription>
+
+      <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+
+      <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+
+
+  <ItemGroup>
+    <SupportedPlatform Include="browser" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="7.0.11" />
```

Comment:
@glatzert das muss man anders einstellen oder?

Reply:
@aysiebel da hat der Aaron recht - schau mal in die Blazor.Components